cookbooks/smoke-tests/attributes/default.rb

```diff
@@ -0,0 +1,18 @@
+# vim: tabstop=2:shiftwidth=2:softtabstop=2 
+default['hadoop_smoke_tests'] = {}
+default['hadoop_smoke_tests']['app_name'] = 'Oozie-Smoke-Test-Coordinator'
+default['hadoop_smoke_tests']['oozie_hosts'] = ['f-bcpc-vm2.bcpc.example.com']
+default['hadoop_smoke_tests']['oozie_user'] = 'ubuntu'
+default['hadoop_smoke_tests']['wf_path'] = 'hdfs://Test-Laptop/user/ubuntu/oozie-smoke-tests/wf'
+default['hadoop_smoke_tests']['carbon-line-receiver'] = "10.0.100.5"
+default['hadoop_smoke_tests']['carbon-line-port'] = 2013
+default['hadoop_smoke_tests']['wf']['co_path'] = 'hdfs://Test-Laptop/user/ubuntu/oozie-smoke-tests/co'
+default['hadoop_smoke_tests']['wf']['krb5_realm'] = 'BCPC.EXAMPLE.COM'
+default['hadoop_smoke_tests']['wf']['rm'] = 'f-bcpc-vm2.bcpc.example.com'
```

Comment:
shouldn't this come from searching the node the way we identify RM nodes when we do HA? What if I decide to pick a different node for RM before I kick off a cluster build?